crates/cli/src/subcommands/identity.rs

```diff
@@ -398,10 +398,6 @@ async fn exec_new(mut config: Config, args: &ArgMatches) -> Result<(), anyhow::E
         query_params,
     )?);
 
-    if let Ok(identity_token) = config.get_default_identity_config(server) {
```

Comment:
For some reason the identity grabbed here is not the correct identity for the server I'm connecting to and I'm getting an invalid identity response. We just shouldn't be supplying auth here at all IMO.

Reply:
any chance there's a `spacetime.toml` in the project directory?